src/reporters/LambdaReporter.ts

```diff
@@ -0,0 +1,104 @@
+import { HttpClient } from "@actions/http-client";
+import { throws } from "assert";
+import { CheckOptions } from "../check";
+import { Result } from "../ComplianceChecker";
+import { Product } from "../model/Product";
+import { Reporter } from "./Reporter";
+
+interface ResultRecord {
+  level: string,
+  rule?: string,
+  check?: string,
+  mandatory?: string,
+  result: string,
+  reason?: string
```

Comment:
<details>
<summary>:warning: <strong>This code contains a statement without a semicolon.</strong> (click for details)</summary>

A possible improvement would be to keep the code consistency and insert the semicolon at the end of the line.  
  
This change would ensure the code follows the current code standard, making it easier for new developers to start without the need to learn a different convention and, thus, improving maintainability and keeping the code consistency.

**[More details](https://confluence.devfactory.com/x/lAsaF)**

## The improvement suggestion in this case
Consider adding the semicolon at the end of the line.  
  
```suggestion  
  reason?: string;
```
</details>

[//]: # (<a id="0490789d-e1e0-473b-be87-076e5fd6f29f" data-finding-id="337251220" href="none" codereview-uuid-reserved-tag data-business-id="55ce652a-40cf-48e1-916a-f8d32a3b8762" data-analyzer="js" />)